quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Start.java

```diff
@@ -63,4 +66,16 @@ public boolean includeRuntime() {
     public String getName() {
         return NAME;
     }
+    
+    public static void fastStart(Picocli picocli, boolean dryRun) {
+        try {
+            Start start = new Start();
+            start.optimizedMixin.optimized = true;
+            start.dryRunMixin.dryRun = dryRun;
+            start.setPicocli(picocli);
+            start.run();
+        } catch (PropertyException | ProfileException e) {
+            picocli.usageException(e.getMessage(), e.getCause());
+        }
+    }
```

Comment:
👍  to this. We won't have to have a special fast start logic which caused e.g. https://github.com/keycloak/keycloak/issues/30380.